corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html

```diff
@@ -53,6 +53,9 @@ <h4 class="modal-title">
                     <p class="help-block" data-bind="visible: appearance() == 'daterange'">
                       {% trans 'In "YYYY-MM-DD to YYYY-MM-DD" format' %}
                     </p>
+                    <p class="help-block" data-bind="visible: isMultiselect">
+                      {% trans 'e.g. 0#,#3#,#5 would select 1st, 4th and 6th choices as default values' %}
```

Comment:
this syntax seems like it is going to trip people up, if it's necessary can't we apply it on the backend and just have the user type `0,3,5`.

The zero indexing might also be a challenge.

Reply:
> this syntax seems like it is going to trip people up, if it's necessary can't we apply it on the backend and just have the user type 0,3,5.

Great idea! I will do that.

> The zero indexing might also be a challenge.

This we already follow for single selects, so changing this for existing apps might be tricky, so I will that as is.